### PR TITLE
Add new alipay plus brands

### PR DESCRIPTION
**Pull request by phieronymus**
Linked to https://github.com/univapaycast/univapay-common/issues/1126

### CODE REVIEW

File: src/resources/common/enums.ts [hunk 1]
```diff
@@ -142,12 +142,32 @@ export enum OnlineBrand {
...
+    KREDIVO = "kredivo",
+
     /**
      * @deprecated Use `ALIPAY_PLUS_ONLINE` instead
      */
     ALIPAY_CONNECT_ONLINE = "alipay_connect_online",
 }
 
+// Alipay+ Offline
+export enum RetailsBrands {
```
Comment (gdp88) [inline src/resources/common/enums.ts:160]: should we go with `RetailBrands` to be consistent with this naming pattern that we use plural only for the last word?

**Review by gdp88 — APPROVED** [review]

**Comment by gdp88** [comment]
minor comment..lgtm!

File: src/resources/common/enums.ts (hunk 1 above)
Comment (phieronymus) [reply src/resources/common/enums.ts]: Indeed, thanks for catching that! 👍

**Comment by gdp88** [comment]
thanx! merging!

**Pull request merged**